### PR TITLE
Pyperclip moved to pyclip

### PR DESCRIPTION
**Pull request by LucasTavaresA**
See: https://pypi.org/project/pyperclip3/

### CODE REVIEW

**Comment by LaurenceWarne** [comment]
Hi, thanks for the contribution!
> Pyperclip moved to pyclip

Is this right? `pip3 install pyperclip` still works for me, and https://pypi.org/project/pyperclip/ is still up n running.  https://github.com/spyoungtech/pyclip appears to me a different project?  Though I'd be open to providing support for it, something like:

```python
try:
    import pyperclip
except ImportError:
    try:
        import pyclip as pyperclip
    except ImportError:
        PYPERCLIP = False
```

What do you think?

**Comment by LucasTavaresA** [comment]
yeah im having doubts from that page i linked as well, but pyperclip is not working on wayland cause it expects $DISPLAY variable which is not used by wayland anymore or never was used.
pyclip instead just separates the wayland logic from the rest of x11, feel free to do what you wish

File: README.md [hunk 1]
```diff
@@ -42,5 +42,11 @@ c.hints.selectors["code"] = [
 pip3 install pyperclip --user
 ```
 
+5. Pyperclip doesn't work on wayland so you can use [pyclip](https://github.com/spyoungtech/pyclip) instead:
```
Comment (LaurenceWarne) [inline README.md]: I think it would make sense to make this part of point 4.

File: README.md [hunk 2]
```diff
@@ -42,5 +42,11 @@ c.hints.selectors["code"] = [
...
+
+```
+pip install pyclip
+```
```
Comment (LaurenceWarne) [inline README.md:49]: Perhaps change to `pip3`, in case `pip` points to python 2 installation?

**Review by LaurenceWarne — COMMENTED** [review]
Just a couple of minor comments, else looks good!

**Review by LaurenceWarne — APPROVED** [review]
lgtm, thanks for this :slightly_smiling_face:

**Pull request merged**